utopian-io/v2.utopian.io
# QEditor Image Sprint

### Verify that

Please provide a test plan so that the reviewers can see how you tested your work and/or the different acceptance criteria described in the task.

### **Test plan**

_Feature / acceptance criteria tested_

* Step 1 - lerna clean , lerna link , lerna bootstrap 
* Step 2 - Visit /en/articles/create and create a new article
* Step 3a - Drag an image file from the filesystem to the Editor pane, drop it.
* Step 3b - Drag several image files from the filesystem to the Editor pane, drop them.
* Step 3c - Drag a non-image file from the filesystem to the Editor pane, drop it.
* Step 4a - Copy an image file from the filesystem and paste it into the Editor pane 
* Step 4b - Copy several image files from the filesystem and paste them into the Editor pane
* Step 4c - Copy a non-image file from the filesystem and paste it into the Editor pane 
* Step 5a - Click the image button, select a file from the filesystem.
* Step 5b - Click the image button, select multiple images.
* Step 5c - Click the image button, select non-image file. 
* Step 6 - Paste rich text (like from your IDE) into the editor
* Step 7 - "UPDATE THE ARTICLE"
* Step 8 - Reload the page (everything should still be there!)

**Expected result:** You should for each case of 3,4 & 5:
- see an image in the HTML Editor
- see a link in the MD code preview
- see an image in the MD render preview
- not be able to upload a non-image (if that is the scope)

**Extra credit**
- Visit http://localhost:8080/en/croppa
- Upload an image, drag it around the circle and zoom with the scroll wheel
- It is wired up to the IPFS endpoint (click the green button)
- It is buttery smooth on mobile
